metasyn/varspec.py

```diff
@@ -64,6 +64,7 @@ def parse(cls, dist_spec: Optional[Union[dict, type[BaseDistribution], BaseDistr
         ------
         TypeError
             If the input has the wrong type and cannot be parsed.
+
```

Comment:
I missed this variation, but this extra line also seems a little bit extraneous? If I look at PEP 257 it doesn't seem to mention anything like that?

Reply:
Your're right, good catch, this was also introduced when I hadn't yet configured ruff properly